src/System.Private.CoreLib/shared/System/Security/SecureString.Unix.cs

```diff
@@ -66,21 +67,21 @@ private void DisposeCore()
             if (_buffer != null && !_buffer.IsInvalid)
             {
                 _buffer.Dispose();
-                _buffer = null;
+                _buffer = null!;
             }
         }
 
         private void ClearCore()
         {
             _decryptedLength = 0;
-            _buffer.Clear();
+            _buffer!.Clear();
```

Comment:
Debug.Assert?

Reply:
You can add a Debug.Assert to _buffer in various places and avoid using `!` that also adds extra validation when in `Debug` mode... Will stop commenting on those.